app/javascript/components/student/SolutionSummary.tsx

```diff
@@ -14,14 +15,31 @@ export type SolutionSummaryLinks = {
 }
 
 export const SolutionSummary = ({
-  iteration,
+  solutionId,
+  iteration: initialIteration,
```

Comment:
I think this should be `latestIteration` not `initialIteration`. `initialIteration` would suggest to me that it's the first iteration of the solution.